api/HTMLAllCollection.json

```diff
@@ -23,10 +23,10 @@
             "version_added": "11"
           },
           "opera": {
-            "version_added": true
+            "version_added": "15"
```

Comment:
http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=8803 in Opera 12.16 shows `document.all` is a `HTMLCollection`, so this seems right.